addons/mail/static/src/components/attachment_viewer/attachment_viewer.js

```diff
@@ -156,12 +151,13 @@ class AttachmentViewer extends Component {
      * @private
      */
     _handleImageLoad() {
+        const refs = this._getRefs();
+        const image = refs[`image_${this.attachmentViewer.attachment.id}`];
```

Comment:
I think we can have a race condition here: if `attachmentViewer` or `attachment` is deleted just before the image is loaded, we might go into the `_handleImageLoad` before the component is destroyed (but models are already deleted).
It would be safer to guard those variables with preemptive `return`.